config/migrations/2024/20240314161912-ensure-mock-account-abb-is-admin.sparql

```diff
@@ -0,0 +1,30 @@
+DELETE {
+  GRAPH ?g {
+    ?mock <http://mu.semte.ch/vocabularies/ext/sessionRole> ?role.
+  }
+  GRAPH ?j {
+    ?person <http://xmlns.com/foaf/0.1/familyName> ?name.
+  }
+}
+INSERT {
+  GRAPH ?g {
+    ?mock <http://mu.semte.ch/vocabularies/ext/sessionRole> "LoketAdmin", "LoketLB-vendorManagementGebruiker".
```

Comment:
Is there a need to add "LoketLB-vendorManagementGebruiker" as a session role for ABB?